apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -69,18 +109,31 @@ export default class LevelFeedbackEntry extends Component {
 
     const seenByStudent = seen_on_feedback_page_at || student_first_visited_at;
 
+    const baseHeight = performance && comment.length > 2 ? 132 : 112;
```

Comment:
Similar here, these feel like magic numbers and probably deserve an explanation.